extensions/sql-database-projects/src/controllers/projectController.ts

```diff
@@ -62,6 +62,12 @@ export class ProjectsController {
 			newProject = await Project.openProject(projectFile.fsPath);
 			this.projects.push(newProject);
 
+			// open any reference projects (don't need to worry about circular dependencies because those aren't allowed)
```

Comment:
good catch, but they are there - we should not crash.

Reply:
it doesn't crash, but it shows the info notification that the project is already open. Should I make it so that this doesn't show if there's a circular reference?

Reply:
this PR doesn't add support for adding project references (that'll come later), but SSDT prevents it

Reply:
added an error message if a circularly referenced project is already open